### PR TITLE
Wrong completion count

### PR DESCRIPTION
**Pull request by KillerX**

### CODE REVIEW

File: backend/user/middleware.go [hunk 1]
```diff
@@ -159,6 +159,7 @@ func NewUserMiddleware(queries *sqlc.Queries, membersClient *members.Client) fun
 			span.AddEvent("User failed to load")
 			userCache.Set(userID, u, cache.WithExpiration(1*time.Minute))
 			ctx.Set(CtxUser, u)
+			return
```
Comment (KillerX) [inline backend/user/middleware.go:162]: This should have been there all along. We bail out if there is an error getting person data, not related to this pr but it slipped in.

File: queries/studies.sql [hunk 2]
```diff
@@ -161,7 +162,7 @@ WITH total AS (SELECT t.lesson_id,
 SELECT total.lesson_id as id, p.id as parent_id
 FROM users.profiles p
          JOIN completed ON completed.profile_id = p.id
-         JOIN total ON total.lesson_id = completed.lesson_id
+         JOIN total ON total.lesson_id >= completed.lesson_id -- In case somethig has been archived later
```
Comment (fredrikved) [inline queries/studies.sql:165]: >= doesn't make sense here on the join or?
Wouldn't there be a lot of rows?

Comment (KillerX) [reply queries/studies.sql:165]: Loooool I put this in a totally wrong place :D. Gimme a moment.

Comment (KillerX) [reply queries/studies.sql]: Fixed.

**Review by fredrikved — APPROVED** [review]

**Pull request merged**